test.py

```diff
@@ -0,0 +1,101 @@
+"""Unit tests for main module."""
+
+import logging
+import json
+import posixpath
+import unittest
+from unittest import mock
+from fastapi.testclient import TestClient
+
+import main
+
+class RoutingTest(unittest.TestCase):
+    """Unit tests for routing and each operation."""
+
+    def setUp(self):
+        patcher_load_config_file = mock.patch("main.load_config_file")
+        patcher_get_client = mock.patch("main.get_client")
+        self.mocked_load_config_file = patcher_load_config_file.start()
+        self.mocked_get_client = patcher_get_client.start()
+        self.addCleanup(patcher_load_config_file.stop)
+        self.addCleanup(patcher_get_client.stop)
+
+    @mock.patch.dict("main.configs",
+                     {"master_path": "master_path/", "repository_path": "repo_path/"})
```

Comment:
I also thought about your suggestion, but I thought it makes us difficult to understand each test function because we cannot see how is mocked at a glance.

So if you are okay, I want to remain it as it is.